tests/unit/notice_normaliser/test_metadata_extractor.py

```diff
@@ -0,0 +1,67 @@
+from ted_sws.domain.model.metadata import ExtractedMetadata
+from ted_sws.metadata_normaliser.services import xpath_extract_data, xpath_extract_attributes
+
+import xml.etree.ElementTree as ET
+
+from ted_sws.metadata_normaliser.services.extract_metadata import MetadataExtractor
+
+
+def test_metadata_extractor(raw_notice):
+    metadata_extractor = MetadataExtractor(notice=raw_notice).extract_metadata()
+
+    extracted_metadata_dict = metadata_extractor.dict()
+
+    assert isinstance(metadata_extractor, ExtractedMetadata)
+    assert extracted_metadata_dict.keys() == ExtractedMetadata.__fields__.keys()
+    assert "067623-2022" in extracted_metadata_dict["notice_publication_number"]
+
+
+def test_metadata_extractor_2016(notice_2016):
+    metadata_extractor = MetadataExtractor(notice=notice_2016).extract_metadata()
+
+    extracted_metadata_dict = metadata_extractor.dict()
+    assert isinstance(metadata_extractor, ExtractedMetadata)
+    assert extracted_metadata_dict.keys() == ExtractedMetadata.__fields__.keys()
+    assert notice_2016.ted_id in extracted_metadata_dict["notice_publication_number"]
+
+
+def test_metadata_extractor_2015(notice_2015):
+    metadata_extractor = MetadataExtractor(notice=notice_2015).extract_metadata()
+
+    extracted_metadata_dict = metadata_extractor.dict()
+    assert isinstance(metadata_extractor, ExtractedMetadata)
+    assert extracted_metadata_dict.keys() == ExtractedMetadata.__fields__.keys()
+    assert notice_2015.ted_id in extracted_metadata_dict["notice_publication_number"]
+
+def test_metadata_extractor_2018(notice_2018):
+    metadata_extractor = MetadataExtractor(notice=notice_2018).extract_metadata()
+
+    extracted_metadata_dict = metadata_extractor.dict()
+    assert isinstance(metadata_extractor, ExtractedMetadata)
+    assert extracted_metadata_dict.keys() == ExtractedMetadata.__fields__.keys()
+    assert notice_2018.ted_id in extracted_metadata_dict["notice_publication_number"]
```

Comment:
why only this key? 

You see, This is the reason why I recommended to list the keys as variables in the "feature file"